### PR TITLE
zsh: add system packages completion paths to fpath

### PR DESCRIPTION
**Pull request by uvNikita**

### CODE REVIEW

File: modules/programs/zsh.nix [hunk 1]
```diff
@@ -256,6 +262,8 @@ in
       home.file."${relToDotDir ".zshrc"}".text = ''
         fpath+="$HOME/.nix-profile/share/zsh/site-functions"
         fpath+="$HOME/.nix-profile/share/zsh/$ZSH_VERSION/functions"
+        fpath+="/run/current-system/sw/share/zsh/site-functions"
+        fpath+="/run/current-system/sw/share/zsh/$ZSH_VERSION/functions"
```
Comment (rycee) [inline modules/programs/zsh.nix]: Would these work for a non-NixOS system?

I note that the zsh module in NixOS [includes](https://github.com/NixOS/nixpkgs/blob/931a0b8be80661902baefb3e7d55403be893e0e6/nixos/modules/programs/zsh/zsh.nix#L162-L165) something like this already.

**Comment by uvNikita** [comment]
Good point, I haven't thought about non-NixOS users. Fixed by using the same code as in zsh module from nixpkgs.

**Comment by rycee** [comment]
Cool! I can't say whether this actually works or not but looks good to me ;-)

**Comment by uvNikita** [comment]
Merged into master: https://github.com/rycee/home-manager/commit/2304c145f342ae98968c1ea295f02c1183539ea3. Thanks for the review :)

**Pull request closed without merging**